app/invocation/invocation_raw_logs_card.tsx

```diff
@@ -92,6 +96,13 @@ export default class RawLogsCardComponent extends React.Component {
                   See more events
                 </div>
               )}
+            {this.props.pageSize &&
+              filteredEvents.length > this.props.pageSize * this.state.numPages &&
+              !!this.state.numPages && (
+                <div className="all" onClick={this.handleAllClicked.bind(this)}>
```

Comment:
nit: since this renders under the same conditions as the above div, you can instead write this as

```
            {this.props.pageSize &&
              filteredEvents.length > this.props.pageSize * this.state.numPages &&
              !!this.state.numPages && (
                <>
                <div className="more" onClick={this.handleMoreClicked.bind(this)}>
                  See more events
                </div>
                <div className="more" onClick={this.handleAllClicked.bind(this)}>
                  See all events
                </div>
                </>
              )}
```

The `<>...</>` is short-hand for `<React.Fragment>...</React.Fragment>` which lets you group together elements without an explicit DOM wrapper such as `<div>`.

See https://reactjs.org/docs/fragments.html

Reply:
Done!